values.yaml

```diff
@@ -802,6 +802,19 @@ connectInject:
   image: null # image for consul-k8s that contains the injector
   default: false # true will inject by default, otherwise requires annotation
 
+  # healthChecks enables propagation of k8s pod health status to Consul service instances.
+  # NOTE: It is highly recommended to enable TLS with this feature because it requires
+  # making calls to Consul clients across the cluster. Without TLS enabled, these calls
+  # could leak ACL tokens.
+  healthChecks:
+    # true has the effect of marking a Consul service instance passing/critical in the event
+    # that the connect-injected pod has passed/failed readiness checks.
+    # Setting this to false will cause the controller to not run and health check status to not be synced.
```

Comment:
```suggestion
    # enables the Consul Health Check controller that syncs the readiness checks on
    # connect-injected services with the agent, to route traffic only to healthy pods.
```

Reply:
This suggestion looks great! nit: could we capitalize `enables` plz. Also, let's remove the last clause: `, to route traffic only to healthy pods` since it makes it sound that the hc controller routes traffic.

Reply:
💯